client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/utils.tsx

```diff
@@ -85,7 +85,22 @@ export const itemVariantColumn = (
   label: 'label.item-variant',
   width: 150,
   Cell: InboundLineItemVariantInputCell,
-  setter: updateDraftLine,
+  setter: patch => {
+    const { packSize, itemVariant } = patch;
+
+    if (itemVariant) {
+      const packaging = itemVariant.packagingVariants.find(
+        p => p.packSize === packSize
+      );
+      updateDraftLine({
+        ...patch,
+        volumePerPack:
+          ((packaging?.volumePerUnit ?? 0) / 1000) * (packSize ?? 1),
```

Comment:
```suggestion
        volumePerPack:
           // itemVariant packaging volume is in L, invoice/stock line volume is in m3, need to convert here
          ((packaging?.volumePerUnit ?? 0) / 1000) * (packSize ?? 1),
```
Nit - some kind of comment here explaining? seeing as different units isn't in the field names?